composer.json

```diff
@@ -19,7 +19,7 @@
         "illuminate/view": "^5.5.0|^6.0",
         "ramsey/uuid": "^3.0",
         "psr/log": "^1.0",
-        "scoutapp/scout-apm-php": "^1.0"
+        "scoutapp/scout-apm-php": "dev-master"
```

Comment:
This must be updated to `^2.0` before this PR is merged (therefore, depends on the release of the PHP lib `2.0.0`)